src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1207,32 +1208,119 @@ void missingRdfTypeOnNamedNode() {
         }
     }
 
+    class BaseExample {
+        Model model;
+        Resource system;
+        ValidateAIF validator = seedlingValidator;
+
+        void setup() {
+            model = createModel();
+            system = makeSystemWithURI(model, getTestSystemUri());
+        }
+
+        void addType(Resource resource, Resource type) {
+            markType(model, getAssertionUri(), resource, type, system, 1d);
+        }
+
+        void testInvalid(String name) {
+            assertAndDump(model, name, validator, false);
```

Comment:
Please update the java doc / comments for assertAndDump() on line 1546. The current wording sounds like the model is dumped for every test. After talking with @ecurley-nextcentury the model is only getting dumped when a test fails or if the FORCE_DUMP flag is set to _true_.

Reply:
The most recent commit added Javadoc to assertAndDump() explaining this.